viz_scripts/plots.py

```diff
@@ -154,7 +119,7 @@ def func(pct, values):
     
     wedges, texts, autotexts = ax.pie(m_values,
                                       labels = m_labels,
-                                      colors=[colours[key] for key in labels],
+                                      colors=[colors[key] for key in labels],
```

Comment:
Ditto

Reply:
Reverted the changes. Reverted it to colours instead of colors.